src/kOS/Function/Misc.cs

```diff
@@ -28,6 +28,51 @@ public override void Execute(SharedObjects shared)
         }
     }
 
+[Function("hudtxt")]
+    public class FunctionHudTxt : FunctionBase
+    {
+
+        public override void Execute(SharedObjects shared)
+
+        {
+            int mirror          = Convert.ToInt32(shared.Cpu.PopValue());
+            string color        = shared.Cpu.PopValue().ToString();
```

Comment:
I like it.
It makes sense and promotes flow in writing scripts.
I might have to lean on your help for this, I'm such pathetic noob.
So, a couple of thoughts.
Firstly, the HTML tag compatibility is inherited from the postScreenMessage function. I know it also responds to the italics tag, but I don't know what else. 
I struggled with the concept of saddling a user with things like the HUD: prefix hardcoded into the mirror to terminal option. I injected the HUD: prefix as a bit of "branding", perhaps self aggrandizing...Just so you know how I understand how some choices are more than simply objective. You guys have really respected other authors with regards to how you extend their plugins etc.
I thought about maybe giving the hudtext function a distinct colour. Originally I thought about limiting it to one screen placement. But as you learn about the extent of the API you start to rationalize more and more freedom. 
What I am getting at is: if we are deciding now and forever what functionality hudtext is going to have, maybe I should invest a few hours to see what's possible. Can we change fonts? Can we underline? Can we change the background cold? Can we change alignment? Can we superscript or subscript? do we really care?
There is a lot to suggest a decision needs to be made to either lock down functions, add the RGBA, or leave it open to whatever it supports.